### ISSUE TITLE
[BUG] Accuracy overlay shows wrong values for Nailgun

### ISSUE THREAD

**Issue by The-Gig** [OpenArena/gamecode#164]
It's possible to bind "+acc" command to a key in order to get an "Accuracy overlay" shown while holding that key.
Example _/bind l "+acc"_ will make the table show when you will be holding the "l" key.

But the percentages shown for _Nailgun_ are clearly wrong. You may fire a single shot at close range and then see a 600% accuracy for that weapon, which makes no sense.

Maybe the problem could be somehow related to the fact that a single Nailgun shot actually fires 15 nails, which reach the target at different times.

Considerations:
- Wondering whether "Accuracy" medal (which is _single player deathmatch_ only, which is earned if you land more than 50% of shots) calculations are affected, too. Harder to test, I fear... By the way, considering Accuracy Overlay does not consider Gauntlet and Proximity Mines, I wonder whether Accuracy Medal considers them or not.
- Shotgun seems to do _not_ have the same issue. Considering that's an hitscan (instant hit) weapon, I guess the code is quite different.
- While doing the testing, be aware that to update the stats shown there, you have to release and push the button again, as _the table is not updated automatically_. IDK if that's on purpose to be able to see the stats of "static" moment instead of ever-changing values or to prevent excessive computational work... or just an overlook. Thinking about it, if the table would have been updated automatically, people would have been able to enable/disable the overlay typing /+acc and /-acc in console.

![shot0247](https://user-images.githubusercontent.com/23258514/173807434-98c5714f-6de9-432d-9a1f-2f1d9784869d.jpg)

**Comment by NeonKnightOA** [comment]
Before tackling this issue, I need to finish developer mode. That way we can take a look at what's actually happening.

**Comment by NeonKnightOA** [comment]
Does the bug still appear in the newer versions?

**Comment by The-Gig** [comment]
Yes, I just shot a close range nailgun in OAX B53 and got 300% accuracy.

**Comment by NeonKnightOA** [comment]
Does it happen with the other weapons as well or solely with the Nailgun? I haven't checked yet the game module (only cgame).

**Comment by The-Gig** [comment]
IIRC, it's just the nailgun.

**Comment by NeonKnightOA** [comment]
I need to know in which situation the bug is triggered and if it happens with these releases:

- Last nightly
- Last 2023 nightly
- b52
- First nightly
- 0.8.8

**Comment by NeonKnightOA** [comment]
I've added D/L links to my last post.
